### ISSUE TITLE
Vaginal prolapse

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#11502]
**Preferred term label:**

**Synonyms**
Vaginal apex prolapse
Vaginal apical prolapse

**Definition (free text, please give PubMed ID)**
PMID: 28306229 and PMID: 32572181 The major clinical criteria of clEDS are (1) skin hyper extensibility, with velvety skin texture and absence of atrophic scarring; (2) generalized joint hypermobility with or without recurrent dislocations (most commonly shoulder and ankle); and (3) easy or spontaneous bruising of skin.
Minor criteria are (1) foot deformities including broad/plump forefoot, brachydactyly with excessive skin, pes planus, hallux valgus, and piezogenic papules; (2) edema in the legs in absence of cardiac failure; (3) mild proximal and distal muscle weakness; (4) axonal polyneuropathy; (5) atrophy of muscles in hands and feet; (6) acrogeric hands, mallet finger(s), clinodactyly, and brachydactyly; and (7) **vaginal/uterus/rectal prolapse**

https://www.msdmanuals.com/home/women-s-health-issues/pelvic-organ-prolapse-pop/pelvic-organ-prolapse-pop?utm_source=chatgpt.com : Prolapse of the vagina

In prolapse of the vagina, the upper part of the vagina drops down into the lower part, so that the vagina turns inside out. The upper part may drop part way through the vagina or all the way through, protruding outside the body and causing total vaginal prolapse.

Total vaginal prolapse may cause pain while sitting or walking. If vaginal tissue protrudes past the opening of the vagina and is exposed to air or rubs on clothing, bleeding, a discharge, sores, or infection may develop. Like prolapse of the uterus, prolapse of the vagina can cause problems with urination. Having a bowel movement may also be difficult.

**Parent term (use hpo.jax.org/app)**
Pelvic organ prolapse  HP:0031607

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Classical-like Ehlers-Danlos syndrome type 1 ORPHA:230839

**Your nano-attribution (ORCID)**
0009-0005-6714-5727

**Comment by pnrobinson** [comment]
Anatomical prolapse with descent of at least one of the vaginal walls to or beyond the vaginal hymen with maximal Valsalva effort with the presence either of bothersome characteristic symptoms, most commonly the sensation of vaginal bulge, or of functional or medical compromise due to prolapse without symptom bother.
PMID: 34191102

See also -- Vaginal hernia HP:0100672 - this is distinct, but the definition needs work and we need to consider the overall hierarchy

**Comment by carolina-f** [comment]
That's the definition of a vaginal hernia; it's not the same as vaginal prolapse. But I agree that the definition should be revised. See issue #11501

![Image](https://github.com/user-attachments/assets/74e9d67f-accd-4ce1-b1ca-b9ed92ebb4a5)

**Comment by pnrobinson** [comment]
@carolina-f researching this a bit, I think that the terms are essentially synonyms:
A pelvic organ prolapse is a hernia of the pelvic region of the female reproductive system. 
Vaginal Prolapse" is the standard medical term for the drooping of organs; "Vaginal Hernia" is a descriptor often used to explain how the tissue is herniating through the vaginal wall.

The existing term [Vaginal hernia (HP:0100672)](https://hpo.jax.org/browse/term/HP:0100672) and its children match the intended definition:
https://www.tgh.org/institutes-and-services/conditions/pelvic-organ-prolapse

I think this term should be renamed
**Pelvic organ prolapse**
Def: Pelvic organ prolapse (POP) refers to the protrusion of one or more female pelvic organs outside the pelvis through the vagina including uterus, bladder and intestines, and causes the pelvic organs to drop downward to the vaginal wall. That is, POP is defined as the anatomical prolapse with descent of at least one of the vaginal walls to or beyond the vaginal hymen with maximal Valsalva effort with the presence either of bothersome characteristic symptoms, most commonly the sensation of vaginal bulge, or of functional or medical compromise due to prolapse without symptom bother.
PMID: 34191102
PMID: 22414610
Syn: Female genital prolapse, Vaginal prolapse, Vaginal hernia

@carolina-f Thoughts?

**Comment by carolina-f** [comment]
I agree with your interpretation that “vaginal hernia” and “pelvic organ prolapse (POP)” are essentially describing the same underlying pathological process, with the former being more of a descriptive term and the latter the standard clinical terminology.

Following your reasoning, I support the proposal to rename the existing term Vaginal hernia (HP:0100672) to Pelvic organ prolapse, while keeping “vaginal hernia” as a synonym.

In addition, I would like to confirm that it would be appropriate to structure the hierarchy so that specific subtypes are represented under this term, including:

Cystocele (fallen bladder)
Rectocele (fallen rectum)
Enterocele (bulging intestine)
Uterine prolapse (fallen uterus)
Vaginal vault prolapse (fallen vagina) (Vaginal apex prolapse, Vaginal apical prolapse)

as described in the Tampa General Hospital and MSD Manual references.

I think this grouping would help ensure consistency with clinical classification and improve clarity for users.

**Comment by pnrobinson** [comment]
@carolina-f merged with existing term HP:0031607, **Pelvic organ prolapse**, and updated definition and synyonyms as above.
